index.bs

```diff
@@ -4269,16 +4210,94 @@ handled on the server side and do not need support from the API specified here.
 
+This specification defines a Web API and a cryptographic peer-entity authentication protocol.
+The [=Web Authentication API=] allows Web developers (i.e., "authors") to utilize the Web Authentication protocol in their
+[=registration=] and [=authentication=] [=ceremonies=].
+The entities comprising the Web Authentication protocol endpoints are user-controlled [=authenticators=] and a [=[RP]=]'s
+computing environment hosting the [=[RP]=]'s web application.
+In this model, the user agent, together with the [=[WAC]=], comprise an intermediary between [=authenticators=] and [=[RPS]=].
+Additionally, [=authenticators=] can [=attestation|attest=] to [=[RPS]=] as to their provenance.
+
+At this time, this specification does not feature detailed security considerations. However, the [[FIDOSecRef]] document provides a security analysis which is overall applicable to this specfication.
+Also, the [[FIDOAuthnrSecReqs]] document suite defines [=authenticator=] security characteristics which are overall applicable for WebAuthn [=authenticators=].
+
+The below subsections comprise the current Web Authentication-specific security considerations.
+
+
 ## Cryptographic Challenges ## {#cryptographic-challenges}
+
 As a cryptographic protocol, Web Authentication is dependent upon randomized challenges
-to avoid replay attacks. Therefore, both {MakePublicKeyCredentialOptions/challenge}}'s 
-and {{PublicKeyCredentialRequestOptions/challenge}}'s value, MUST be randomly generated
-by the [=Relying Party=] in an environment they trust (e.g., on the server-side), and the challenge in the client's
-response must match what was generated. This should be done in a fashion that does not rely
-upon a client's behavior; e.g.: the Relying Party should store the challenge temporarily
+to avoid replay attacks. Therefore, both {{MakePublicKeyCredentialOptions/challenge}}'s
+and {{PublicKeyCredentialRequestOptions/challenge}}'s value MUST be randomly generated
+by [=[RPS]=] in an environment they trust (e.g., on the server-side), and the
+returned challenge value in the client's
+response MUST match what was generated. This SHOULD be done in a fashion that does not rely
+upon a client's behavior; e.g.: the Relying Party SHOULD store the challenge temporarily
 until the operation is complete. Tolerating a mismatch will compromise the security
 of the protocol.
 
+## Attestation Security Considerations ## {#sec-attestation-security-considerations}
+
+### Attestation Certificate Hierarchy ### {#cert-hierarchy}
+
+A 3-tier hierarchy for attestation certificates is RECOMMENDED (i.e., Attestation Root, Attestation Issuing CA, Attestation
+Certificate). It is also RECOMMENDED that for each WebAuthn Authenticator device line (i.e., model), a separate issuing CA is
+used to help facilitate isolating problems with a specific version of a device.
+
+If the attestation root certificate is not dedicated to a single WebAuthn Authenticator device line (i.e., AAGUID), the AAGUID
```

Comment:
The previous paragraph recommends a 3-level certificate chain with a new intermediate for each model. This paragraph then implicitly recommends a 2-level chain with a separate _root_ for each model. These seem to be contradictory.

I would tend to recommend a 2-level chain (and would suggest putting the AAGUID in the certificate in all cases): I expect the FIDO MDS to be the metadata source for attestation information and, if a whole model (i.e. AAGUID) needs to be "revoked", then the status of that AAGUID as a whole can be changed in the MDS. (I'm presuming that AAGUID will become a key in the MDS in the future since CTAP2 is a FIDO spec.)

Reply:
Ok, so perhaps this ought to be a separate issue, which we submit and address separately from this PR. This is because I only moved the above text to this new section, and @rlin1 needs to weigh-in since AFAIK he wrote the above and has been the primary editor of the Metadata Statement and Metadata Service specs. 

WDYT?